endpoint/common.php

```diff
@@ -27,6 +27,7 @@ function load_config()
 	if(array_key_exists('DB_DBNAME',$GLOBALS)) $config['dbname'] = $GLOBALS['DB_DBNAME'];
 	if(array_key_exists('MEMCACHE_HOST',$GLOBALS)) $config['memcache_host'] = $GLOBALS['MEMCACHE_HOST'];
 	if(array_key_exists('MEMCACHE_PORT',$GLOBALS)) $config['memcache_port'] = $GLOBALS['MEMCACHE_PORT'];
+    	if(array_key_exists('TOPIC',$GLOBALS)) $config['topic'] = $GLOBALS['TOPIC'];
 
```

Comment:
Think you should remove this line otherwise the test will pass even if the value is not in the configuration file?

Reply:
Done